packages/airnode-validator/src/ois/ois.ts

```diff
@@ -219,4 +306,6 @@ export const baseOisSchema = z.object({
   endpoints: z.array(endpointSchema),
 });
 
-export const oisSchema = baseOisSchema.superRefine(ensureSingleParameterUsagePerEndpoint);
+export const oisSchema = baseOisSchema
+  .superRefine(ensureSingleParameterUsagePerEndpoint)
+  .superRefine(ensureEndpointAndApiSpecificationParamsMatch);
```

Comment:
Nit: could rename the refine functions from `ensure`... to `validate`... if we want to keep the function names similar everywhere

Reply:
That is a good point and left me thinking about this.

I'd not replace this, because they are not synonyms. For example, you can't say `ensureSecuritySchemesReferences` instead of `validateSecuritySchemesReferences`. I like the `ensure` a bit more since it forces you to be more specific which is nicer. But I understand it might not be easy (or worth it) to name the functions this way all the time.

Reply:
That's good reasoning and I agree, thanks for explaining. Let's keep them   :+1: